### DISCUSSION TITLE
GEM Weather elements on grid

### DISCUSSION THREAD

**Discussion by phillipseamore** [open-meteo/open-meteo#245] [General]
This is extremely interesting: https://eccc-msc.github.io/open-data/msc-data/nwp_gdps/readme_gdps-datamart-alpha_en/

It's also available for regional and HDRPS.

These are basically all the elements needed to generate a descriptive forecast. It seems to have been running for a couple of years (I guess it's at least used operationally in CA) but is still defined as alpha.

Technical note: https://collaboration.cmc.ec.gc.ca/cmc/cmoi/product_guide/docs/tech_notes/technote_weong-hrdps_e.pdf

The gribs are on a different server than NWP outputs. Many of the variables are discrete classes so probably shouldn't be interpolated.

**Comment by patrick-zippenfenig** [comment]
Thanks for sharing! I noticed the `WEonG` on the production server as well. Maybe they are getting closer to a final version.

The precipitation probability (POP) calculation is interesting. The use of Burr distribution for POP is new to me. Adding POP from GEM WEonG might be worth the effort

Unfortunately, other meteorological fields like temperature, wind or humidity are still raw model output. MOS or  bias corrected fields would be very nice.

**Comment by phillipseamore** [comment]
According to the tech note only windgust, winddirection, temperature and dewpoint are raw from the deterministic outputs, all the other ones are derived with some even checking neighboring points out to more than 100km. I see a lot of potential in this for both symbol determination and summary text creation - a poor man's area forecast if you will 😄

And perhaps that's what this should be, there's "current", "hourly" and "daily" now in the API, could this be "area"?
